vdm-io/Joomla-Component-Builder
# Few improvments as fixes

Pull Request for Issue gh-212 gh-217, gh-216.

### Summary of Changes
- Added (Linked To) back in related areas, so we can see where what is linked. 
- Added ajax fields to the custom admin view.
- Added some backward compatibility to stylesheets and scripts being added.
- Fixed  no access lang string. 
- Added the site CSS area to Joomla Component view.

### Testing Instructions
- Open admin_views, custom_admin_views, site_views, fields, dynamic_gets, libraries to see if the linked to values are correct.
- See if the ajax fields in the custom admin works.
- check the new compatibility check for style sheets and scripts are behaving correctly.
- Is the language string for no access loading.
- Add some css (to site) in the joomla_component area

### Expected result
- All linked relationships must show
- Be able to add ajax to the custom admin view directly.
- backward compatibility with Joomla 3.7 and lower.
- able to add global css to front-end

### Actual result
None of the expected results were possible before.
